recipes/draco/all/conanfile.py

```diff
@@ -126,4 +125,9 @@ def package(self):
         self.copy(pattern="*.dll", dst="bin", src=build_bin_dir, keep_path=False)
 
     def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "Draco"
+        self.cpp_info.names["cmake_find_package_multi"] = "Draco"
+        self.cpp_info.names["pkg_config"] = "Draco"
```

Comment:
Mmhh I think that pkg_config name I gave is wrong. Draco generates `draco.pc` file with `Name` attribute `Draco`, but what is important is the file name, not `Name` attribute in pc file, right?

Reply:
I think so. What matters is whether `pkg-config` will find the package.